### DISCUSSION TITLE
Github download page is confusing for users

### DISCUSSION THREAD

**Discussion by d-damien** [rustdesk/rustdesk#5411] [General]
Hi. I just had a screen sharing with a client who's a newbie in IT, and the Github download page, with all its architectures listing, was a bit scary to them. Although archive.org didn't record it, the previous website had a user-friendly page with big OS logos, and I think it made it clearer (at least for end users). Could we possibly have that page back even if links just point to Github ?

This being a support software, I believe the discussion is relevant.

Regards.

**Comment by m1ga** [comment]
And also the "scam warning" makes it very strange. Do you really think a scammer would point you to the official page to download the files? They would just put a version on their own servers. 
So normal users who just want to download the file will see the warning in a Github release page and it makes me not want to download that file at all :smile:

**Reply by xlionjuan** [reply]
#1675

**Reply by m1ga** [reply]
https://github.com/rustdesk/rustdesk/discussions/3988
